### PR TITLE
fixed click when clicking on date in month when drag and drop active.…

### PR DESCRIPTION
**Pull request by vasilevich**
Please see issuse:
https://github.com/intljusticemission/react-big-calendar/issues/587#issuecomment-341447496

this is a patch,

### CODE REVIEW

File: src/addons/dragAndDrop/styles.less [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 .rbc-addons-dnd {
   .rbc-row-content {
-    pointer-events: none;
```
Comment (tobiasandersen) [inline src/addons/dragAndDrop/styles.less]: I agree that `pointer-events: none` is a bit problematic here, but removing this introduces new problems when doing DnD. Could it be added only when dragging is active?

File: src/addons/dragAndDrop/styles.less [hunk 2]
```diff
@@ -12,10 +11,10 @@
 
   .rbc-addons-dnd-over {
     background-color: rgba(
-      red(@date-selection-bg-color),
-      green(@date-selection-bg-color),
-      blue(@date-selection-bg-color),
-      .3
+            red(@date-selection-bg-color),
```
Comment (tobiasandersen) [inline src/addons/dragAndDrop/styles.less]: Why the extra indentation?

Comment (vasilevich) [reply src/addons/dragAndDrop/styles.less]: probably by accident.

File: src/addons/dragAndDrop/styles.less (hunk 1 above)
Comment (vasilevich) [reply src/addons/dragAndDrop/styles.less]: can you specify which problems occur? I didn't notice any so far.

Comment (tobiasandersen) [reply src/addons/dragAndDrop/styles.less]: Part of the day slot in the month view becomes undroppable:

demo

Comment (vasilevich) [reply src/addons/dragAndDrop/styles.less]: ahh, I see, I'll look into this shortly with the latest calendar update.

**Comment by arecvlohe** [comment]
@vasilevich How is this coming along? Need any help?

**Comment by vasilevich** [comment]
@arecvlohe hi sorry about the delay, I just been very busy, if you would like you can try to implement this, I haven't done much about it ever since, but I will get to it once I can.

**Comment by jquense** [comment]
sorry we didn't get to merge this! Happy to look again if you want resend it with conflicts addressed

**Pull request closed without merging**